proto/prehog/v1alpha/teleport.proto

```diff
@@ -193,6 +239,24 @@ message UIDiscoverResourceSelectionEvent {
   DiscoverStepStatus status = 3;
 }
 
+// UIDiscoverIntegrationAWSOIDCConnectEvent is emitted when a user is finished with the step
+// that asks user to setup aws integration or select from a list of existing
+// aws integrations.
+message UIDiscoverIntegrationAWSOIDCConnectEvent {
```

Comment:
Same as v12:

I suppose you want those here, so we are piggybacking on the backport to bring the protos as well?

(No objections, just clarifying.)

Reply:
Same as v12:

It's just the latest version, I figured it was harmless enough.